.travis.yml

```diff
@@ -2,22 +2,13 @@ language: c
 
 sudo: required
 
-matrix:
-  include:
-    - os: linux
-      env: BUILD_TYPE=astyle
-    - os: linux
-      compiler: gcc
-    - os: linux
-      compiler: clang
-    - os: osx
-      compiler: gcc
-    - os: osx
-      compiler: clang
-
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ./ci/travis/before_install_darwin ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./ci/travis/before_install_linux ; fi
+env:
+  matrix:
+  - BUILD_TYPE=astyle
```

Comment:
nitpick: Maybe we can replace BUILD_TYPE with JOB since not all the functions presume to build something.

Reply:
I think this i more like a naming preference. 
Each BUILD_TYPE env from matrix determines what should be run in the Build Job.
I used this name to keep consistency with other repos where TravisCI is integrated.